libraries/AP_RangeFinder/AP_RangeFinder_Backend.h

```diff
@@ -33,7 +33,7 @@ class AP_RangeFinder_Backend
     virtual void update() = 0;
     virtual void init_serial(uint8_t serial_instance) {};
 
-    virtual void handle_msg(const mavlink_message_t &msg) { return; }
+    virtual void handle_msg(const mavlink_distance_sensor_t &packet) { return; }
```

Comment:
I'm happy with this part of the change.  We should rename the "handle_msg" to "handle_distance_sensor_msg" though.